### PR TITLE
push code task manage users

### PR DESCRIPTION
**Pull request by khaccuong2k**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by khaccuong2k** [comment]
ready

**Comment by khaccuong2k** [comment]
ready

File: app/Repositories/BaseRepository.php [hunk 1]
```diff
@@ -49,9 +52,14 @@ public function all()
      */
     public function findOrFail($id)
     {
-        $result = $this->model->findOrFail($id);
+        $find = $this->model->findOrFail($id);
+        if (!$find) {
+            Log::debug("Id not found");
```
Comment (quoctd-0607) [inline app/Repositories/BaseRepository.php:58]: e nên dùng try cache xử lý trường hợp này

**Comment by khaccuong2k** [comment]
ready

**Comment by khaccuong2k** [comment]
ready

**Comment by khaccuong2k** [comment]
ready

**Pull request merged**